### PR TITLE
Update to Wordpress Coding Standards, Implement CircleCI

### PR DESCRIPTION
**Pull request by ethan-schaffer**
This PR brings the WordPress coding standards to the plugin. It also includes a config file to run the PHPCS by the CircleCI tool.

### CODE REVIEW

File: includes/class-genesis-tabs-widget.php [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+			/* Custom loop */
+			$tabbed_posts = new WP_Query(
+				array(
+					'cat'       => $cat,
+					'showposts' => 1,
+					'orderby'   => 'date',
+					'order'     => 'DESC',
+				)
+			);
+			if ( $tabbed_posts->have_posts() ) :
```
Comment (rfmeier) [inline includes/class-genesis-tabs-widget.php]: I am not sure why phpcs didn't pick this up, but would you mind switching the clause enclosures to brackets?

**from** 
```
if ( something ) :
    do something
endif;
```

to
```
if ( something ) {
    do something
}
```

File: includes/class-genesis-tabs-widget.php [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+							else :
+								the_content( esc_html( $instance['more_text'] ) );
+							endif;
+
+										endif;
+
+					echo '</div><!--end post_class()-->' . "\n\n";
+
+				endwhile;
+endif;
```
Comment (rfmeier) [inline includes/class-genesis-tabs-widget.php]: There seem to be a few areas the indentations are off.

File: includes/class-genesis-tabs-widget.php [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+	/**
+	 * Widget Output
+	 *
+	 * @param array $args Args.
+	 * @param array $instance Instance.
+	 */
+	public function widget( $args, $instance ) {
+		extract( $args );
+
+		/* defaults */
```
Comment (rfmeier) [inline includes/class-genesis-tabs-widget.php]: Inline comments should be double-slash, capitalized, period at the end. Semi-sentence structure.
```
// The comment.
$some_variable = 1;
```

File: languages/genesis-tabs.pot [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+msgid ""
+msgstr ""
+"Project-Id-Version: Genesis Tabs 0.9.4\n"
+"Report-Msgid-Bugs-To: StudioPress <translations@studiopress.com>\n"
+"POT-Creation-Date: 2019-06-17 3:33:00+00:00\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2018-MO-DA HO:MI+ZONE\n"
+"Last-Translator: StudioPress <translations@studiopress.com>\n"
```
Comment (dreamwhisper) [inline languages/genesis-tabs.pot]:
```suggestion
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```

These should be the person translating.

File: languages/genesis-tabs.pot [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+"Language-Team: English <translations@studiopress.com>\n"
```
Comment (dreamwhisper) [inline languages/genesis-tabs.pot]:
```suggestion
"Language-Team: LANGUAGE <LL@li.org>\n"
```

**Pull request merged**